results.tex

```diff
@@ -29,116 +29,116 @@ \subsubsection{Machine Learning algorithms}
 			\item \textbf{Generative adversarial networks (GANs)}
 				consist of two separate networks, one trying to recognize objects from a dataset and one trying to create new data in an attempt to 'fool' the other network into thinking the data is legit.\cite{Li:2013:CAL:2463372.2465801}
 			\item \textbf{Radial Basis Function (RBF) networks}
 				\cite{rbf}
 			\item \textbf{Self-organizing maps (SOMs) / self-organizing feature maps (SOFMs)}
-				are neural networks that learn by unsupervised \textbf{competitive learning}, in which nodes compete for access to specific inputs, causing the nodes to become highly specialized, which reduces redundancy. Iterations effectively move the map closer to the training data, which is where it gets its name from. Some subtypes include time adaptive self-organizing map (TASOM), Binary Tree TASOM (BTASOM) and growing self-organizing map (GSOM)
+				are neural networks that learn by unsupervised \textbf{competitive learning} in which nodes compete for access to specific inputs, causing the nodes to become highly specialized, which reduces redundancy. The iterations effectively move the map closer to the training data, which is the cause of its name. Some subtypes include the Time Adaptive Self-Organizing Map (TASOM), Binary Tree TASOM (BTASOM) and Growing Self-Organizing map (GSOM)
 			\item \textbf{Stochastic neural networks}
-				make use of stochastic transfer functions or weights, which makes it possible to escape a local minimum. 
+				make use of stochastic transfer functions or weights which makes it possible to escape a local minimum. 
 			% 		boltzmann machine
 		\end{itemize}
-		Neural networks trained in many different ways, like using Generic algorithms. The most common approach, especially when talking about Deep neural networks, is using a process called \textbf{backpropagation};
+		Neural networks are trained in many different ways, like by using genetic algorithms. The most common approach, especially when talking about Deep neural networks, is using a process called \textbf{backpropagation};
 		\begin{itemize}
 			\item Present a training sample
 			\item Calculate the error in each output neuron, how much lower or higher the output must be adjusted to match the desired output
 			\item Calculate the gradient
 			\item For every layer, adjust the weights and biases based on the gradient
 			\item Repeat
 		\end{itemize}
-	\item \textbf{Autoencoders}
-		are a type of neural network that trained specifically to encode and decode data. Because autoencoders are trained to also decode, the encoded version of the data can be seen as a dimensional reduction of the data
+	\item \textbf{Auto-encoders}
+		are a type of neural network that trained specifically to encode and decode data. Because auto-encoders are trained to perform decoding apart from encoding, the encoded version of the data can be seen as a dimensional reduction of the data
```

Comment:
apart -> separately

Reply:
don't agree with this one, see http://context.reverso.net/translation/dutch-english/afgezien+van

Reply:
"Afgezien van" zou ook niet kloppen hier,  het vertaalt eerder naar "afzonderlijk van"